File: C#/ToyLanguage/ToyLanguage/Model/Statements/SkipStatement.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ToyLanguage.Interfaces;

namespace ToyLanguage.Model.Statements
{
    [Serializable]
    class SkipStatement : IMyStatement
    {
        public ProgramState execute(ProgramState programState)
        {
            //do nothing
            return null;
        }

        public string MyToString()
        {
            return "Skip";
        }

    }
}

File: Java/src/model/Statements/SkipStatement.java
package model.Statements;

import Exceptions.DivideByZeroException;
import Exceptions.ValueNotFoundException;
import interfaces.*;
import model.ProgramState;

/**
 * Created by Lucian on 11/10/2015.
 */
public class SkipStatement implements IStatement{

    @Override
    public String toString() {
        return "Skip";
    }

    @Override
    public ProgramState execute(ProgramState programState) throws DivideByZeroException, ValueNotFoundException {
        //do nothing
        return null;
    }
}
